testing/detectors/EMC/workflows.desc

```diff
@@ -1,2 +1,2 @@
-emc-qcall-ctf-5: "O2PDPSuite" reco,5,5,"SHMSIZE=64000000000 testing/detectors/EMC/runEMCRawToDigitsRecoPilelineQCAlllocalCTF.sh"
-emc-qcall-ctf-9: "O2PDPSuite" reco,9,9,"SHMSIZE=64000000000 testing/detectors/EMC/runEMCRawToDigitsRecoPilelineQCAlllocalCTF.sh"
\ No newline at end of file
+emc-qcall-ctf-5: "O2PDPSuite" reco,5,5,"FILEWORKDIR=/home/epn/odc/files WORKFLOW_PARAMETERS=CTF,GPU,EVENT_DISPLAY WORKFLOW_DETECTORS=EMC SHMSIZE=64000000000 testing/detectors/EMC/runEMCRawToDigitsRecoPilelineQCAlllocalCTF.sh"
```

Comment:
`FILEWORKDIR` `INRAWCHANNAME` `CTF_DIR` are set by the EPN, please don't overwrite them here.
I should actually add a comment to the documentation.

Reply:
no problem, it wasn't documented... :)